study.md

```diff
@@ -41,7 +41,7 @@ In your own words, define what the responsibilities of the model layer are in
 Rails.
 
 ```md
-<!-- your answer here -->
+<!-- Models are use to outline the logic of the app develop, in this outline validations of data are included as well as relationships between databases. -->
```

Comment:
The model layer represents data from a database as Ruby objects.  These objects act as an interface between Rails and the database to read, write, update, and destroy data.